src/esp/nav/PathFinder.cpp

```diff
@@ -14,9 +14,10 @@
 #include <Magnum/EigenIntegration/Integration.h>
 
 #include <Corrade/Containers/Optional.h>
+#include <math.h>
 
 #include <cstdio>
-#define _USE_MATH_DEFINES
+#define USE_MATH_DEFINES
```

Comment:
Why this change? What's the macro for?

Why is `<math.h>` included again, if we have `<cmath>` below?